Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    django-user
```

Comment:
The `adduser` command is creating a user without a home directory and password, which is good for security. However, ensure that any application processes that require a home directory are adjusted accordingly.